### ISSUE TITLE
Trigger app.on('text' in group

### ISSUE THREAD

**Issue by indmind** [telegraf/telegraf#361]
is there any way to trigger `app.on('text'` in a group without replying bot message?

**Comment by 1valdis** [comment]
You should disable Privacy mode for your bot in Botfather. See [Telegram Bot API documentation](https://core.telegram.org/bots#privacy-mode).

**Comment by indmind** [comment]
I've already disabled the Privacy Mode, and still, it doesn't catch the message if not replying

**Comment by 1valdis** [comment]
I personally had same issue until I re-added my bot to the group. After that it was able to see all the messages. Did you try that?

**Comment by indmind** [comment]
yeah, it's working now, you right, I need to re-add the bot to the group thanks :smile:
